static/js/admin/src/statusMonitor/DepositMonitor.js

```diff
@@ -105,18 +105,24 @@ define('DepositMonitor', [ 'jquery', 'jquery-ui', 'underscore', 'AbstractStatusM
 		if ("currentJob" in results) {
 			var currentJob = results.jobs[results.currentJob];
 		}
-		
+
 		if ("jobs" in results) {
+			const jobsWithCounts = [
+				"ValidateFileAvailabilityJob",
+				"VirusScanJob",
+				"FixityCheckJob",
+				"ExtractTechnicalMetadataJob",
+				"IngestContentObjectsJob"
+			];
+
 			for (var index in results.jobs) {
 				var job = results.jobs[index];
 				var jobName = job.name.substring(job.name.lastIndexOf(".") + 1);
 				
 				job["shortName"] = jobName;
-				
-				if (jobName == "IngestContentObjectsJob" && "total" in job) {
-					var completion = results.ingestedObjects? results.ingestedObjects : 0;
-					completion += " / " + job.total;
-					job["completion"] = completion;
+
+				if (jobsWithCounts.includes(jobName) && "total" in job) {
```

Comment:
Is there any reason to restrict which jobs we allow counts on? Seems like anything that has `num` set and greater than zero should be displayed. Also, I think its okay to display num even if total isn't set, just display without the second half